demisto_sdk/commands/common/logger.py

```diff
@@ -6,6 +6,12 @@
 
 from demisto_sdk.commands.common.content_constant_paths import CONTENT_PATH
 
+# logger: logging.Logger = logging.getLogger("")
```

Comment:
please add `TID251`, add `logging` to ruff's [config](https://beta.ruff.rs/docs/rules/banned-api/), and supplying a relevant message for future devs, to make sure everybody uses our logger

Reply:
https://github.com/demisto/demisto-sdk/pull/2917/commits/3a48ab2de708286612fdc2f6066e19d132a711a2